pkg/logs/launchers/container/tailerfactory/file_test.go

```diff
@@ -317,7 +318,7 @@ func TestMakeK8sSource(t *testing.T) {
 			require.Equal(t, wildcard, child.Config.Path)
 			require.Equal(t, "src", child.Config.Source)
 			require.Equal(t, "svc", child.Config.Service)
-			require.Equal(t, []string{"tag!"}, child.Config.Tags)
+			assert.EqualValues(t, config.StringSliceField{"tag!"}, child.Config.Tags)
```

Comment:
Could we convert `StringSliceField` to `[]string` instead to avoid an extra dependencies to the config ?

Would this work ?
```suggestion
			assert.EqualValues(t, []string{"tag!"}, []string(child.Config.Tags))
```

Same for the other tests.

Reply:
This is the only place where an extra dependency is added, and I have made the suggested fix so that I can remove the dependency. It doesn't work in so easily in the inventorychecks_test, but an extra dependency isn't added there, so I just kept that as-is.